### ISSUE TITLE
Not sure where to put these tags

### ISSUE THREAD

**Issue by jdawsonadams** [jdawsonadams/responsive-typography#1]
All my screenshots are accepted (even though the one for 1440px probably shouldn't be), but markbot wants to see the "body" selector, the "article p" selector, and the "article ol" selector. I've already used those selectors under the media query tags but I'm assuming markbot wants to see them on my own. I'm not sure what I would put in them if all my screenshots are accepted because I don't want to screw everything up. 

@thomasjbradley

**Comment by thomasjbradley** [comment]
Yeah, those selectors need to be outside the media queries too. We need to remove their top margins and set their bottom margins to match the line height. So outside the media query their bottom margin will be `1.3rem`.
